orttraining/orttraining/training_api/training_session.h

```diff
@@ -29,7 +29,8 @@ class TrainingSession {
                   const SessionOptions& session_options,
                   const std::vector<std::shared_ptr<IExecutionProvider>>& providers,
                   CheckpointState* state,
-                  const ModelIdentifiers& model_identifiers);
+                  const ModelIdentifiers& model_identifiers,
+                  gsl::span<OrtCustomOpDomain* const> custom_op_domains);
```

Comment:
shall we give a default value too for training_session as we did for module and optimizer? 

```suggestion
                  gsl::span<OrtCustomOpDomain* const> custom_op_domains = gsl::span<OrtCustomOpDomain* const>());
```